pkgs/by-name/ap/aprx/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+, perl
+}:
+
+stdenv.mkDerivation {
+  pname = "aprx";
+  version = "2.9.1-unstable-2021-09-21";
+
+  src = fetchFromGitHub {
+    owner = "PhirePhly";
+    repo = "aprx";
+    rev = "2c84448fe6d897980234961a87ee4c1d4fad69ec";
+    sha256 = "sha256-01PB7FaG8GmPm1U15/3g1CfQwdYmf3ThZFdVh2zUAl4=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ perl ];
+
+  CFLAGS = "-fcommon -O2";
+
+  configureFlags = [ "--with-erlangstorage" ];
```

Comment:
```suggestion
  configureFlags = [
    (lib.withFeature true "erlangstorage")
  ];
```

Reply:
huh?  what's the point of this?  pulling in another lib function a future reader has to look up seems like adding complexity for no gain.

Reply:
Opting to keep this as-is